Session01/Session01/Program.cs

```diff
@@ -6,31 +6,50 @@
 
 namespace Session01
 {
-    public struct Animal
+    class Animal
     {
-        public String nume;
-        public String sunet;
+        public String name;
+        public String sound;
 
+        public void getSounds()
+        {
```

Comment:
Method name does not correspond to what it actually does:
1. does handle sounds, but sound
2. get means it will return something, but instead it executes an action